Gemfile

```diff
@@ -2,8 +2,8 @@ source 'https://rubygems.org'
 
 gem 'jekyll'
 gem 'jekyll-redirect-from'
-gem 'uswds-jekyll', '~> 4.0'
-gem 'kramdown-parser-gfm'
+gem 'uswds-jekyll', '~> 5.0'
+gem 'mini_racer' # for autoprefixer
```

Comment:
Making this unnecessary in the future by adding to uswds-jekyll: https://github.com/18F/uswds-jekyll/pull/216